lib/launcher.test.js

```diff
@@ -213,6 +220,8 @@ describe('lib/launcher', () => {
       it('waits for the config to be removed and resolves', async () => {
         const promise = stopDaemon(resolver, config);
 
+        await new Promise((resolve) => setTimeout(resolve));
```

Comment:
Since process.kill now done in async function we have to wait for microtasks queue to drain before checking if function were called

This is the best option I've found

Reply:
I usually go with either `await Promise.resolve()` (one micro task) or `await new Promise(setImmediate)` (chain of micro tasks) 🤓

Reply:
unfortunatelly, both options do not work, I think we can leave it as is if you agree